AEPCore/Sources/rules/TokenFinder.swift

```diff
@@ -68,7 +68,7 @@ class TokenFinder: Traversable {
         case TOKEN_KEY_TIMESTAMP_ISO8601_NO_COLON:
             return now.getISO8601DateNoColon()
         case TOKEN_KEY_TIMESTAMP_ISO8601:
```

Comment:
nit: Can you rename this constant to more appropriate name?

Reply:
I renamed it to `TOKEN_KEY_TIMESTAMP_ISO8601_UTC_MILLISECONDS`